config/zones.json

```diff
@@ -33,8 +33,8 @@
     "capacity": {
       "biomass": 1,
       "hydro": 0,
-      "solar": 1883,
-      "wind": 1
+      "solar": 2705,
+      "wind": 0
```

Comment:
Oh no, what happened to the [one wind turbine](https://u.ae/en/information-and-services/environment-and-energy/water-and-energy/types-of-energy-sources/wind-energy)? 😢